products/ocp4/profiles/pci-dss.profile

```diff
@@ -0,0 +1,18 @@
+documentation_complete: true
+
+metadata:
+    SMEs:
+        - JAORMX
+        - jhrozek
+        - mrogers950
+
+reference: https://www.pcisecuritystandards.org/documents/PCI_DSS_v3-2-1.pdf
+
+title: 'PCI-DSS v3.2.1 Control Baseline for Red Hat OpenShift Container Platform 4'
+
+description: |-
+    Ensures PCI-DSS v3.2.1 security configuration settings are applied.
+
```

Comment:
is `platform: ocp4` implied from the path?

Reply:
It's probably infered from product.yml. But I added it anyway now that you suggested it. Explicit is better than implicit.